### DISCUSSION TITLE
Token duration is 25 minutes error in /finances/v0/financialEvents

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/selling-partner-api-models#3421] [General]
Getting below error in /finances/v0/financialEvents API, When requesting with Next Token. Can anyone explain what happens here?

```
Getting below error in /finances/v0/financialEvents API, When requesting with Next Token. Can anyone explain what happens here?

"error":[
{
"code":"InvalidInput",
"message":"Token is not valid, token duration is 25 minutes.",
"details":""
}
];
```

**Comment by tomaszhanc** [comment]
How much time did you wait between the first and the second request? Was it more than 25 minutes?

**Reply by Nilanth** [reply]
Yes @tomaszhanc, More than 25 minutes

**Reply by tomaszhanc** [reply]
So that's probably it. The token is valid for only 25 minutes. You need to request the next page before those 25 minutes.

**Comment by sungolivia** [comment]
@Nilanth  did you get a chance to answer @tomaszhanc's question?
